soweego/linker/classifiers.py

```diff
@@ -109,10 +212,20 @@ def _fit(
         self,
         feature_vectors: pd.Series,
         answers: pd.Series = None,
-        batch_size: int = constants.BATCH_SIZE,
-        epochs: int = constants.EPOCHS,
+        batch_size: int = None,
+        epochs: int = None,
         validation_split: float = constants.VALIDATION_SPLIT,
     ) -> None:
+
+        # if batch size or epochs have not been provided as arguments, and
+        # the current instance has them as attributes, then use those. If not
+        # then use the defaults defined in constants
+        if batch_size is None:
+            batch_size = self.batch_size
```

Comment:
Can you make an example of how this can happen?

Reply:
The `_fit` method is currently only called by the `recordlinkage.KerasAdapter` when training, and it won't pass these parameters. 

The reason they're there is so we can pass parameters to the fit method. Although currently this functionality is not used. Do you think we should remove it (remove the parameters to `_fit` and simply use the ones defined when initializing the NN-classifier)?

Reply:
I think we can leave it as is: we may want to leverage this functionality in the future.
And it's also a sanity check on how `KerasClassifier` behaves.